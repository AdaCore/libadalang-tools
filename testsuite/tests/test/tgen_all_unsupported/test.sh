gnattest -P test --gen-test-vectors
