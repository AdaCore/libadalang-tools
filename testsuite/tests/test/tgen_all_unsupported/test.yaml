description:
    Test that TGen does not crash when there are no supported subprogram for
    testcase generation.

driver: gnattest_driver
control:
    - [XFAIL, 'x86', 'Marshalling not working for 32bits (UB03-008)']
