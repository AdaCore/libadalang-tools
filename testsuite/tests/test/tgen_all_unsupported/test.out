No subprogram supported for test case generation.
Units remaining: 2     Units remaining: 1     
